feg/gateway/services/s8_proxy/servicers/proto_conversions.go

```diff
@@ -190,3 +190,27 @@ func handleFTEID(fteidIE *ie.IE) (*protos.Fteid, uint8, error) {
 	}
 	return fteid, interfaceType, nil
 }
+
+func handlePDNAddressAllocation(paaIE *ie.IE) (*protos.PdnAddressAllocation, protos.PDNType, error) {
+	pdnTypeIE, err := paaIE.PDNType()
+	if err != nil {
+		return nil, 0, err
+	}
+	var pdnType protos.PDNType
+	var paa protos.PdnAddressAllocation
+	switch pdnTypeIE {
+	case gtpv2.PDNTypeIPv4:
+		pdnType = protos.PDNType_IPV4
+		paa.Ipv4Address = paaIE.MustIPv4().String()
+	case gtpv2.PDNTypeIPv6:
+		pdnType = protos.PDNType_IPV6
+		paa.Ipv6Address = paaIE.MustIPv6().String()
+	case gtpv2.PDNTypeIPv4v6:
+		pdnType = protos.PDNType_IPV4V6
+		paa.Ipv6Address = paaIE.MustIPv6().String()
+		paa.Ipv6Address = paaIE.MustIPv6().String()
+	case gtpv2.PDNTypeNonIP:
+		pdnType = protos.PDNType_NonIP
+	}
+	return &paa, pdnType, nil
+}
```

Comment:
same as before, conversion could be direct since GRPC values and GTP values are the same. But I want to keep both implementation separated.